src/mistral_common/protocol/instruct/messages.py

```diff
@@ -142,6 +144,101 @@ def from_openai(cls, openai_chunk: Dict[str, Union[str, Dict[str, str]]]) -> "Im
         return cls.model_validate({"image_url": openai_chunk["image_url"]})
 
 
+class RawAudio(MistralBase):
+    r"""Base64 encoded audio data.
+
+    This class represents raw audio data encoded in base64 format.
+
+    Attributes:
+        data: The base64 encoded audio data, which can be a string or bytes.
+        format: The format of the audio data.
+
+    Examples:
+        >>> audio = RawAudio(data="base64_encoded_audio_data", format="mp3")
+    """
+
+    data: Union[str, bytes]
+    format: str
+
+    @classmethod
+    def from_audio(cls, audio: Audio) -> "RawAudio":
+        """Creates a RawAudio instance from an Audio object.
+
+        Args:
+            audio: An Audio object containing audio data, format, and duration.
+
+        Returns:
+            An AudioChunk instance initialized with the audio data.
+        """
+        format = audio.format
+        data = audio.to_base64(format)
+
+        return cls(data=data, format=format)
+
+    @validator("format")
+    def should_not_be_empty(cls, v: str) -> str:
+        if v not in EXPECTED_FORMAT_VALUES:
+            raise ValueError(f"`format` should be one of {EXPECTED_FORMAT_VALUES}. Got: {v}`")
```

Comment:
Think there is a ValidationError from Pydantic. (low prio)

```suggestion
            raise ValidationError(f"`format` should be one of {EXPECTED_FORMAT_VALUES}. Got: {v}`")
```